sigma-signature-library/sigma_signature/pysigma.py

```diff
@@ -1,6 +1,10 @@
 from . import signatures
 from . import parser
 from yaml.composer import ComposerError
+import logging
+logger = logging.getLogger('pysigma logger')
```

Comment:
Another small thing. While not required, logger names are often in a classpath format.
https://docs.python.org/3/library/logging.html#logging.getLogger
Using the logger name 'pysigma' is fine on its own, you don't need to include ' logger'
We probably don't want our loggers hard coded to info in production code.

Reply:
👍 
https://github.com/CybercentreCanada/assemblyline-service-sigma/commit/01c717a8178278c061474c63fcffd6cabef98a08

Reply:
Generally speaking, we don't want to be calling setLevel, addHandler or any similar methods on a logger in the global scope.

For seeing the logs during tests, that should be done in the test harness (pytest setup, the run script you are using, whatever is appropriate for the type of test you are doing). Tossing it at the top of the file like this is fine as a temporary thing for debugging when there isn't anywhere else to put it, but you should take it back out when you are done.

If certain logs should be seen during a normal production run, the log level and handler should be configured as part of that entry point (initialization code, the `if __name__ == '__main__':` block, `__main__.py` file or somewhere similar)